manifests/c/CheatBreaker/CheatBreaker/2.4.2/CheatBreaker.CheatBreaker.installer.yaml

```
@@ -0,0 +1,18 @@
+# Created with Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: CheatBreaker.CheatBreaker
+PackageVersion: 2.4.2
+InstallerLocale: en-US
+InstallerType: nullsoft
+Scope: user
+UpgradeBehavior: install
+ReleaseDate: 2023-10-19
+InstallerSwitches:
+  SilentWithProgress: 
+Installers:
+- Architecture: x86

```

```suggestion
- Architecture: x64
```
Our `x86` validations are performed on 32-bit OSes.